src/status_im/ui/screens/wallet/accounts/styles.cljs

```diff
@@ -1,5 +1,36 @@
 (ns status-im.ui.screens.wallet.accounts.styles
-  (:require [status-im.ui.components.colors :as colors]))
+  (:require [status-im.ui.components.colors :as colors]
+            [status-im.ui.components.reanimated :as reanimated]
+            [status-im.utils.platform :as platform]))
+
+(def ^:const tabbar-height 56)
+;; TODO(Ferossgp): get layout size of total-value
+(def ^:const value-height (+ 40 22 8))
+(def ^:const scroll-offset value-height)
+(def ^:const minimized-value-line-height 28)
+
+(defn topbar [{:keys [value offset inset-top]}]
+  (merge
+   {:flex-direction     :row
+    :padding-horizontal 8
+    :background-color   colors/white
+    :padding-top        inset-top}
+   (when platform/android?
+     {:elevation (reanimated/interpolate value
+                                         {:inputRange  [0 offset]
+                                          :outputRange [0 4]
+                                          :extrapolate (:clamp reanimated/extrapolate)})})
+   (when platform/ios?
+     {:shadow-opacity (reanimated/interpolate value
+                                              {:inputRange  [0 offset]
+                                               :outputRange [0 1]
+                                               :extrapolate (:clamp reanimated/extrapolate)})
+      :shadow-radius  16
+      :z-index        2
+      :shadow-color   (if (colors/dark?)
```

Comment:
@flexsurfer @Ferossgp do we have some global map for dark/light colors or we are fine with having `if (colors/dark?)` everywhere?

Reply:
we have a map

Reply:
Everywhere in case of shadows, we have conditionals, and these colors are not defined in colors namespace. I actually took it from here https://github.com/status-im/status-react/pull/10311/files#diff-690bfb052fe61fb1bffec61a578ec30fR43-R45 should I add a new color to be used in shadows?

Reply:
@Ferossgp how shadows are different form other colors and why it can't live in that map?

Reply:
They are not different, but they were never used there and I followed the style of the current code. Everywhere in code where we use shadows, the color does not use a variable but uses the color value directly.